components/selection/README.md

```diff
@@ -2,146 +2,328 @@
 
 The selection components (`d2l-selection-action`, `d2l-selection-input`, `d2l-selection-select-all`, `d2l-selection-summary`, `d2l-selection-action`) are low-level components that provide the ability to create selection interfaces with select-all and bulk-action behaviours.
 
+<!-- docs: start hidden content -->
 ![Selection](./screenshots/selection-multiple.png?raw=true)
 
 ![Selection](./screenshots/selection-single.png?raw=true)
+<!-- docs: end hidden content -->
 
 ## SelectionMixin
 
-The selection components above work with a component that extends the `SelectionMixin`, which acts like a controller for the checkboxes, radios, actions, etc. The `d2l-selection-input` component must be placed _within_ the component that extends the `SelectionMixin`.  The other selection components may also be placed inside the `SelectionMixin` component, or in the same DOM scope with the `selection-for` attribute set to the id of that component.
-
-The `d2l-list` already extends `SelectionMixin` and should always be used for lists, however a custom selection control can be easily defined to enable the use of these selection controls in different semantic contexts or radically different layouts. The `SelectionMixin` defines the `selection-single` attribute that consumers can specify for single selection behaviour.
```

Comment:
This was just moved below the example so that the example appeared closer to the top of the page